### PR TITLE
Policy RL sets learning rate using `K.set_value`. Fixes slowdown.

### PR DESCRIPTION
**Pull request by wrongu**
Previously, `optimizer.lr = K.abs(optimizer.lr) * ...` added another call to `K.abs()` in the computation graph each iteration, slowing down training as time went on.

### CODE REVIEW

File: AlphaGo/training/reinforcement_policy_trainer.py [hunk 1]
```diff
@@ -78,7 +78,7 @@ def run_n_games(optimizer, learner, opponent, num_games, mock_states=[]):
     # Train on each game's results, setting the learning rate negative to 'unlearn' positions from
     # games where the learner lost.
     for (st_tensor, mv_tensor, won) in zip(state_tensors, move_tensors, learner_won):
-        optimizer.lr = K.abs(optimizer.lr) * (+1 if won else -1)
+        K.set_value(optimizer.lr, abs(lr) * (+1 if won else -1))
```
Comment (MaMiFreak) [inline AlphaGo/training/reinforcement_policy_trainer.py]: why use the `abs()` here instead of when passing it to `run_n_games`?

File: AlphaGo/training/reinforcement_policy_trainer.py [hunk 2]
```diff
@@ -208,7 +208,7 @@ def save_metadata():
 
         # Run games (and learn from results). Keep track of the win ratio vs each opponent over
         # time.
-        win_ratio = run_n_games(optimizer, player, opponent, args.game_batch)
+        win_ratio = run_n_games(optimizer, args.learning_rate, player, opponent, args.game_batch)
```
Comment (MaMiFreak) [inline AlphaGo/training/reinforcement_policy_trainer.py]: `win_ratio = run_n_games(optimizer, abs(args.learning_rate), player, opponent, args.game_batch)`

File: AlphaGo/training/reinforcement_policy_trainer.py (hunk 1 above)
Comment (wrongu) [inline AlphaGo/training/reinforcement_policy_trainer.py]: I figured that it wouldn't add any overhead to call `abs` here, and that it is slightly better from a readability standpoint since it puts `abs` on the same line as `+1` & `-1`.
